### PR TITLE
Update criteria.md

### PR DESCRIPTION
**Pull request by Zephyr3755**
Update of some factors that are not properly placed and additional of specific guidelines

### CODE REVIEW

**Comment by Emptiest** [comment]
Zephyr asked me to tag the issue #134

**Comment by Illuminati-CRAZ** [comment]
image

**Review by Illuminati-CRAZ — CHANGES_REQUESTED** [review]
Portions of these changes come from osu!wiki, and so somewhere on the document it should say that portions of this come from it

File: en-US/ranking/criteria.md [hunk 1]
```diff
@@ -20,8 +38,22 @@ The following pertains to metadata that your maps contain such as song artists,
 * **The artist and title must be absolutely correct.**
 * **The artist and title must be romanized.** However, non-romanized characters are allowed to be used in source and tags.
 * **The artist, title and source must not repeat in tags.** However, romanizing or translating the source in tags is encouraged, in case it's relevant.
+* **Source must not a website name. It should be filled with album, mixtape, game name, etc.** However, if it comes from youtube, soundcloud and such, source should remain empty.
```
Comment (Illuminati-CRAZ) [inline en-US/ranking/criteria.md:41]:
```suggestion
* **The source should be an album, mixtape, game, show, etc. that the song comes from.** Youtube, Soundcloud, etc., are not valid sources. This field may be left empty if there is no relevant source.
```

File: en-US/ranking/criteria.md [hunk 2]
```diff
@@ -20,8 +38,22 @@ The following pertains to metadata that your maps contain such as song artists,
...
 * **Tags must contain the genre of the song.**
-* **Every difficulty must have a difficulty name.**
+* **Tags must be related to the beatmap and not misleading for search results.** Tags describing the beatmap's style, song content are considered to be related to the beatmap.
```
Comment (Illuminati-CRAZ) [inline en-US/ranking/criteria.md:43]:
```suggestion
* **Tags must be related to the map and not misleading for search results.**
```

File: en-US/ranking/criteria.md [hunk 3]
```diff
@@ -20,8 +38,22 @@ The following pertains to metadata that your maps contain such as song artists,
...
+* **Guest difficulty creators must be added to the tags of a beatmapset.** Only a registered username ingame or website are valid (using discord name is discouraged due how interchangeable that is)
```
Comment (Illuminati-CRAZ) [inline en-US/ranking/criteria.md:44]:
```suggestion
* **Guest difficulty creators must be added to the tags of a mapset.** 
```

File: en-US/ranking/criteria.md [hunk 4]
```diff
@@ -20,8 +38,22 @@ The following pertains to metadata that your maps contain such as song artists,
...
+
+### Metadata Standardisation
+
+* **Any form of `feat.`, `feat`, `ft.`, etc. must be written as `feat.` when used as an indicator for an artist featured in the song.**
```
Comment (Illuminati-CRAZ) [inline en-US/ranking/criteria.md:48]:
```suggestion
* **Any form of `feat.`, `feat`, `ft.`, etc. must be written as `feat.` when used as an indicator for an artist featured in the song.** Artists featured in the song should be part of the artist field, and not the title field.
```

File: en-US/ranking/criteria.md [hunk 5]
```diff
@@ -30,7 +62,7 @@ The following pertains to media files such as background images and audio files
 * **The resolution of background images must be at least 1280x720** and of exceptionally high quality. The file size of a background image must not exceed 4 MB.
 * **The mapset must contain only one audio file.** Multiple song file mapsets are not eligible to be ranked.
 * **MP3 is the only allowed audio file format.**
-* **The maximum bitrate allowed for audio files is 192kbps.**
+* **The maximum bitrate allowed for audio files is 192kbps.** Check with http://spectro.enpts.com/download.php for the waveforms, refer to this https://homedjstudio.com/audio-bitrates-formats/ article about hertz (hz).
```
Comment (Illuminati-CRAZ) [inline en-US/ranking/criteria.md:65]:
```suggestion
* **The maximum bitrate allowed for audio files is 192kbps.**
```

File: en-US/ranking/criteria.md [hunk 6]
```diff
@@ -46,70 +78,106 @@ The following requirements are for the individual maps themselves.
 * **The maps must be at least 45 seconds long.**
 * **The maps must be timed as accurately as possible.** Timing points must not be used in place of scroll velocity changes.
 * **The maps must have a song select preview point that sufficently compliments the song.** It's usually placed before the chorus or at the beginning of a section. A solid preview point attracts more players to try the map.
+* **BPM lines should not be used abusively for decorations or color changes.** This could be a confusing to players to play and redundant as in filesize.
```
Comment (Illuminati-CRAZ) [inline en-US/ranking/criteria.md:81]:
```suggestion
```

File: en-US/ranking/criteria.md [hunk 7]
```diff
@@ -46,70 +78,106 @@ The following requirements are for the individual maps themselves.
...
+* **Objects must be snapped to timeline ticks according to snaps.** Use any custom snap as accurate as possible to the song.
```
Comment (Illuminati-CRAZ) [inline en-US/ranking/criteria.md:82]: Change to a guideline

```suggestion
* **Notes should generally be snapped according to a common beat snap divisor.** Custom beat snap divisors may be used to more accurately time a note. 
```

File: en-US/ranking/criteria.md [hunk 8]
```diff
@@ -46,70 +78,106 @@ The following requirements are for the individual maps themselves.
...
 
 ### Difficulty Spread
 
 The following pertains to the spread of difficulty each mapset must follow.
+Rhythm-related rules and guidelines apply to approximately 160 BPM beatmaps. If your song is drastically faster or slower, some variables might be different, as detailed in Scaling BPM on the Ranking Criteria.
```
Comment (Illuminati-CRAZ) [inline en-US/ranking/criteria.md:87]: Move to section actually talking about difficulty specifics

```suggestion
Rhythm-related guidelines apply to approximately 160 BPM beatmaps.
```

File: en-US/ranking/criteria.md [hunk 9]
```diff
@@ -46,70 +78,106 @@ The following requirements are for the individual maps themselves.
...
+
+##### Multi-Game Mode Sets
+
+Furthermore, **each game mode is treated separately.** If you are creating a mapset for both 4 Keys and 7 Keys, you must have a difficulty spread for both game modes.
+
+**If you have two game modes, you must preface each difficulty with either "4K" or "7K"** (Example: "7K Insane").
+
+## Guidelines
+
+The following are guidelines to consider when creating your mapset and submitting it for ranked status. **These are not mandatory and it can be broken under special circumstances.** Our intentions with this are to allow mappers to be creative with the types of maps they produce along with clear representation and not getting overmapped. As such, these guidelines are for bridges to increase your chances of getting your mapset ranked.
```
Comment (Illuminati-CRAZ) [inline en-US/ranking/criteria.md:115]: deleted text seems unnecessary to me

```suggestion
The following are guidelines to consider when creating your mapset and submitting it for ranked status. **These are not mandatory and can be broken under special circumstances.** 
```

File: en-US/ranking/criteria.md [hunk 10]
```diff
@@ -46,70 +78,106 @@ The following requirements are for the individual maps themselves.
...
 * **Beginner**
      * For players who are being exposed to rhythm games for the first time.
      * Notes should typically be on every other beat in most cases and once every beat to portray emphasis in the music.
-     * Jumps are suitable for this difficulty when emphasizing important sounds in the music.
-     * Scroll velocity changes must not be used at this difficulty level for any other purposes than scroll velocity normalization.
+     * Chords are suitable for this difficulty when emphasizing important sounds in the music. However, it must be used sparingly.
+     *  **Note density should consist of mostly 2/1, occasional 1/1.**
+     * **Avoid chords with more than 2 notes.** However, using a chord at beginning or the end of long note is allowed to use. This applies to 4 and 7 keymode.
+     * **Long notes should be held for at least 2/1.** Faster timing release could be difficult for players of this level. Smaller gap between any note require moderate level of finger independence that players of this difficulty barely handle.
+     * **Scroll velocity changes must not be used at this difficulty level for any other purposes than scroll velocity normalization.**
```
Comment (Illuminati-CRAZ) [inline en-US/ranking/criteria.md:126]:
```suggestion
     * **Scroll velocity changes should not be used at this difficulty level for any other purposes than scroll velocity normalization.**
```

File: en-US/ranking/criteria.md [hunk 11]
```diff
@@ -46,70 +78,106 @@ The following requirements are for the individual maps themselves.
...
+##### Multi-Game Mode Sets
+
+Furthermore, **each game mode is treated separately.** If you are creating a mapset for both 4 Keys and 7 Keys, you must have a difficulty spread for both game modes.
+
+**If you have two game modes, you must preface each difficulty with either "4K" or "7K"** (Example: "7K Insane").
+
+## Guidelines
+
+The following are guidelines to consider when creating your mapset and submitting it for ranked status. **These are not mandatory and it can be broken under special circumstances.** Our intentions with this are to allow mappers to be creative with the types of maps they produce along with clear representation and not getting overmapped. As such, these guidelines are for bridges to increase your chances of getting your mapset ranked.
+
```
Comment (Illuminati-CRAZ) [inline en-US/ranking/criteria.md:116]: I'm just gonna search for every mention of the word "must" here and replace it with "should" lol

File: en-US/ranking/criteria.md [hunk 12]
```diff
@@ -46,70 +78,106 @@ The following requirements are for the individual maps themselves.
...
+##### Difficulty Specifics
+
 * **Beginner**
      * For players who are being exposed to rhythm games for the first time.
      * Notes should typically be on every other beat in most cases and once every beat to portray emphasis in the music.
-     * Jumps are suitable for this difficulty when emphasizing important sounds in the music.
-     * Scroll velocity changes must not be used at this difficulty level for any other purposes than scroll velocity normalization.
+     * Chords are suitable for this difficulty when emphasizing important sounds in the music. However, it must be used sparingly.
```
Comment (Illuminati-CRAZ) [inline en-US/ranking/criteria.md:122]:
```suggestion
     * Chords are suitable for this difficulty when emphasizing important sounds in the music. However, they should be used sparingly.
```

File: en-US/ranking/criteria.md [hunk 13]
```diff
@@ -46,70 +78,106 @@ The following requirements are for the individual maps themselves.
...
-     * Jumps can be used more freely but sparingly, as players at this level may still have trouble with them.
-     * Scroll velocity changes must not be used at this difficulty level for any other purposes than scroll velocity normalization.
+     * One beat streams are generally comfortable to these players.
+     * **Avoid using more than 7 consecutive 1/2 notes.**
+     * **Note density should consist of mostly 1/1, occasional 1/2.**
+     * **Note snappings of consecutive 1/3 and above should not be used.** Slower rhythms are more appropriate for this difficulty level.
+     * **Chords can be used occasionally, as players at this level still have trouble with them.**
+     * **Avoid using chords during consecutive 1/2 note patterns.** However, using a chord to begin or to end a pattern is fine.
+     * **Avoid chords with more than 2 notes.** This includes long note patterns that involve notes in more than two columns. This applies to 4 and 7 keymode.
+     * **Scroll velocity changes must not be used at this difficulty level for any other purposes than scroll velocity normalization.
```
Comment (Illuminati-CRAZ) [inline en-US/ranking/criteria.md:139]:
```suggestion
     * **Scroll velocity changes should not be used at this difficulty level for any other purposes than scroll velocity normalization.
```

File: en-US/ranking/criteria.md (hunk 6 above)
Comment (Illuminati-CRAZ) [reply en-US/ranking/criteria.md:81]: Or move to guidelines:

"Timing points should properly reflect the song. When used improperly, they can cause confusion to the player. "

**Review by Illuminati-CRAZ — CHANGES_REQUESTED** [review]
Portions of these changes come from osu!wiki, and so somewhere on the document it should say that portions of this come from it

File: en-US/ranking/criteria.md (hunk 1 above)
Comment (Zephyr3755) [reply en-US/ranking/criteria.md:41]: im not against this but kind of ehhhhhhh, the vagueness is ok but you know, most people (that still confused to rc) still did it. youtube, soundcloud, bandcamp etc are valid, but not good to be added on source field. would you suggest another wording for me?

File: en-US/ranking/criteria.md (hunk 6 above)
Comment (Zephyr3755) [reply en-US/ranking/criteria.md:81]: moved is ok

File: en-US/ranking/criteria.md (hunk 1 above)
Comment (Illuminati-CRAZ) [reply en-US/ranking/criteria.md:41]: idk this seems decently clear to me

**Pull request closed without merging**